### PR TITLE
fix(endpoints) disable healthcheck `PUT` apis in hybrid mode

### PR DESCRIPTION
**Pull request by mayocream**
### Summary

Remove healthy APIs in hybrid mode.

Fix FT-2953

### CODE REVIEW

**Review by bungle — APPROVED** [review]

**Comment by bungle** [comment]
@mayocream I have approved, just one question: what about `dbless`?

**Comment by mayocream** [comment]
> @mayocream I have approved, just one question: what about `dbless`?

Just discussed with @bungle offline that the current API reference docs are correct for dbless.

| API  | DB-less compatible | Hybrid compatible |
| ------------- | ------------- | ------------- |
| `/upstreams/{upstreams}/targets/{targets}/healthy`  |  NO  | NO |
| `/upstreams/{upstreams}/targets/{targets}/unhealthy`  | NO  | NO |
| `/upstreams/{upstreams}/targets/{targets}/{address}/healthy` | YES | NO |
| `/upstreams/{upstreams}/targets/{targets}/{address}/unhealthy` |  YES | NO |

**Review by dndx — APPROVED** [review]

File: kong/api/routes/upstreams.lua [hunk 1]
```diff
@@ -206,42 +206,48 @@ return {
...
-  ["/upstreams/:upstreams/targets/:targets/unhealthy"] = {
     PUT = function(self, db)
-      return set_target_health(self, db, false)
+      return target_endpoint(self, db, update_target_cb)
     end,
   },
+}
 
-  ["/upstreams/:upstreams/targets/:targets/:address/healthy"] = {
+-- upstream targets' htealthcheck management is not available in the hybrid mode
```
Comment (bungle) [inline kong/api/routes/upstreams.lua:225]:
```suggestion
-- upstream targets' healthcheck management is not available in the hybrid mode
```

Comment (dndx) [reply kong/api/routes/upstreams.lua]: Nice catch!

**Review by dndx — APPROVED** [review]

**Pull request merged**